src/server.rs

```diff
@@ -146,15 +145,41 @@ struct ServerStats {
     /// The count of errors writing to cache.
     pub cache_write_errors: u64,
     /// The number of successful cache writes.
-    pub cache_writes: u32,
+    pub cache_writes: u64,
```

Comment:
I can't say I have strong feelings either way. I was mostly just curious to make sure I wasn't missing something important about the patch.